backend/controllers/taskController.js

```diff
@@ -2,6 +2,10 @@ const asyncHandler = require("express-async-handler");
 const Task = require("../models/userTasks");
```

Comment:
For all the routes here, we should also include a return for status code 5XX in the event that something were to go wrong with the database where we can't fetch any data.

I believe some examples with errors handling can be found here:
https://mongoosejs.com/docs/api.html#model_Model.find (This is just an example with using Model.find)

Reply:
Fixed: Changes are reflected in recent commits